.github/workflows/integration.yml

```
@@ -16,8 +16,6 @@ jobs:
         - ubuntu-latest
         ansible-version:
         - git+https://github.com/ansible/ansible.git@devel
```

Try using:

```suggestion
        - git+https://github.com/ansible/ansible.git@temp-2.10-devel
```

devel hasn't had any updates for a few weeks. It's possible this has been fixed already in `temp-2.10-devel`

After Monday `devel` will be the same as temp-2.10-devel